pkg/utils/runtime/utils.go

```diff
@@ -58,11 +44,11 @@ func (c *runtime) IsDebug() bool {
 }
 
 func (c *runtime) IsLive() bool {
-	return c.check()
+	return true
```

Comment:
Looks strange to me it always returns `true`, why?

Reply:
As long as the web server is running, why should it return that it is not live ?

Reply:
There is a scenario (usually due to firewall policies) where the Kyverno pod is running just fine but fails to receive any admission requests.

We used to check the annotation on the lease object to verify that Kyverno was able to receive the admission request. This was removed in this PR and I'm wondering if it'd bring any value since it always return true?

Reply:
But why should we return that the pod is not live ? Restarting the pod is not going to help in this scenario right ?

Reply:
I've checked the latest code and only found this annotation `kyverno.io/last-request-time` that Kyverno adds to the lease object. We used to have another annotation indicating whether webhook is active or not (can receive admission request).

I may need your help understanding the latest health check system - Kyverno register a verify webhook that continuously send admission request if there's no activity happening on the cluster, and it updates this annotation on the `lease` object to store the last-seen admission request. Is this still the case? If so, how do we leverage this annotation to report Kyverno actual status?

Reply:
> I've checked the latest code and only found this annotation kyverno.io/last-request-time that Kyverno adds to the lease object. We used to have another annotation indicating whether webhook is active or not (can receive admission request).

I don't remember about the other annotation, how was it used to drive the webhook configuration ?

> I may need your help understanding the latest health check system - Kyverno register a verify webhook that continuously send admission request if there's no activity happening on the cluster, and it updates this annotation on the lease object to store the last-seen admission request. Is this still the case? If so, how do we leverage this annotation to report Kyverno actual status?

This refactor uses a different lease than the one for leader election. Using the same lease has issues because updates will conflict between leader election and health check.

Setting the timestamp is now done in the webhook, if the admission request fails, the timestamp won't get updated. Previously it was set by kyverno so the annotation would have been updated with or without going through the webhook.

I guess that if we detect kyverno is not healthy we should remove webhook configs and set policies state to not ready ?

Reply:
>I don't remember about the other annotation, how was it used to drive the webhook configuration ?

I recall we used to set the value of that annotation to `inactive`, but I cound't find anything in the doc :(

>I guess that if we detect kyverno is not healthy we should remove webhook configs and set policies state to not ready ?

Yes good point! Now that we have `status.ready` for policies, we should set `ready=false` if Kyverno is not able to receive/serve admission requests.

Reply:
Ok, I think we're good then :)

Reply:
Do we set policy to not ready now?

Reply:
Sure

Reply:
```
	} else {
		c.recordPolicyState(config.MutatingWebhookConfigurationName)
	}
```
This is when kyverno is not healthy, all policies will be set to not ready after that.

Reply:
Great, thanks!